### PR TITLE
fix advanced vector search for 1.2

### PR DESCRIPTION
**Pull request by nan-wang**
fix advanced vector search for 1.2

### CODE REVIEW

**Review by rutujasurve94 — DISMISSED** [review]
LGTM 👍

File: advanced-vector-search/app.py [hunk 1]
```diff
@@ -143,10 +113,11 @@ def _get_evaluation_results(evaluation_results: dict, resp):
 
         with Flow.load_config('flow-query.yml') as flow:
             with TimeContext(f'QPS: query with {len(query_input)}', logger=flow.logger):
-                flow.search(input_fn=query_input, request_size=request_size,
+                flow.search(inputs=query_input, request_size=request_size,
                             on_done=get_evaluation_results,
                             top_k=top_k)
 
+        print(f'evaluation: {list(evaluation_results)}')
```
Comment (jacobowitz) [inline advanced-vector-search/app.py:120]: maybe its better to use flow.logger instead of a print?

**Review by jacobowitz — COMMENTED** [review]
One question regarding printing/logging, otherwise looks good to me :)

**Review by cristianmtr — APPROVED** [review]

**Review by Yongxuanzhang — APPROVED** [review]
LGTM. I assume YAML files will be updated in a separate PR?

**Pull request merged**